frontend/sass/_data-driven-onboarding.scss

```diff
@@ -60,6 +60,10 @@
     .jf-ddo-card {
         p.subtitle {
             margin-bottom: 0.25em;
+            .jf-registration-warning {
```

Comment:
Tahnee wanted extra spacing under the "registration warning" part of the intro card. Since we cannot nest another `p` inside this `p` element, I had to make this custom class to add the spacing she wanted.

Reply:
Oh funkadelic.  I guess another option might be to make the parent `<p>`'s into `<div>`'s?  Eh, we can revisit later if we continue to make more CSS changes like this one.

Reply:
You know, another option might be to add some code that turns all indicators that are React fragments into `<p class="subtitle">`s but have anything that's _not_ a fragment just be embedded directly.  That way this warning could just be a `<div>` while all the others could still be paragraphs... I guess it might be worth considering if we add more exceptions like this.